indico/web/templates/forms/permissions_widget.html

```diff
@@ -10,7 +10,7 @@ <h3>Permissions</h3>
                 <a href="#" class="i-button label text-color outline add-entry-decoration">
                     {%- trans %}Add{% endtrans -%}
                 </a>
-                <a class="i-button js-add-user-group">{% trans %}User / Group{% endtrans %}</a>
+                <span id='js-add-user-group'></span>
```

Comment:
```suggestion
                <span id="js-add-user-group"></span>
```

we usually use single quotes in html attributes